LayoutKit/Views/BatchUpdates.swift

```diff
@@ -0,0 +1,69 @@
+// Copyright 2016 LinkedIn Corp.
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import Foundation
+
+
+/// A set of updates to apply to a `ReloadableView`.
+public struct BatchUpdates {
+    public var insertItemsAtIndexPaths = [NSIndexPath]()
```

Comment:
Alternatively `itemInserts`, `sectionInserts`, `itemDeletes`, `sectionDeletes`

Do you prefer past tense?
`insertedItems` vs `insertItems`